app/ts/background/metadataUtils.ts

```diff
@@ -220,15 +223,24 @@ export async function retrieveEnsLabelHashes(events: EnrichedEthereumEvents) {
 	})
 }
 
-export const getAndCacheEnsNodeHash = async (ethereumClientService: EthereumClientService, ensNameHash: EthereumBytes32) => {
-	const currentHashes = await getEnsNodeHashes()
+const addNewEnsNameEntry = async (name: string) => {
+	const [label, _] = name.split('.')
```

Comment:
Omit unused variable (underscore)
```suggestion
	const [label] = name.split('.')
```